build.toml

```diff
@@ -1,6 +1,6 @@
 [archive]
 build_output_dir = 'Built'
-archive_location = '\\nirvana\temp\vs_cds\scan_engine_custom_device'
+archive_location = '\\us-aus-hilbuild\builds\niveristandadd-ons\archive\vs_cds\scan_engine_custom_device'
```

Comment:
My build machine does not have access to \\us-aus-hilbuild and if it did, could possibly overwrite your old builds with this one. This needs to stay at nirvana\temp in the ni-veristand-cds repo.